mm2src/coins/eth.rs

```diff
@@ -59,6 +59,10 @@ use web3::types::{Action as TraceAction, BlockId, BlockNumber, Bytes, CallReques
 use web3::{self, Web3};
 use web3_transport::{EthFeeHistoryNamespace, Web3Transport, Web3TransportNode};
 
+use crate::utxo::utxo_builder::UtxoConfError;
+use crate::utxo::utxo_common::{CheckPaymentSentError, ExtractSecretError, SendRawTxError, ValidatePaymentError};
```

Comment:
`eth.rs` has not to be dependent on `utxo.rs`. Someday `eth` and `utxo` will be separate crates.
Consider moving `CheckPaymentSentError, ExtractSecretError, SendRawTxError, ValidatePaymentError` to a common module. `lp_coins.rs` is too `tight`, so I think you can add `coin_errors`, and move all errors from `lp_coins.rs` to it.

Reply:
ok good idea. Thanks